combat-trainer.lic

```diff
@@ -16,6 +16,8 @@ class SetupProcess
     echo("  @stance_override: #{@stance_override}") if $debug_mode_ct
     @priority_defense = settings.priority_defense
     echo("  @priority_defense: #{@priority_defense}") if $debug_mode_ct
+    @priority_weapons = settings.priority_weapons
```

Comment:
Do you have a character YAML that uses this setting you would be willing to upload?

When someone adds a new setting, I like to have an example of it being used (for others to use as a reference). It's not necessary for merging, though.

Reply:
Sure, happy to upload my character setup yaml

Reply:
Make sure to comment out anything you don't want shared publicly. Safe rooms, etc.

Reply:
I decided to add it to the sample files. If I add my character setup, will it then be auto sync'ed through `dependency` in case I have local changes I would like to keep?